python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -276,6 +279,7 @@ def __get__(self, obj, cls):
 
         # If obj is None then the getter was called on the class so return the Item
         if obj is None:
+            name = _getName(self)
```

Comment:
1. This is repeated from a few lines before
2. Why don't we just do `self._name`?